manifester/settings.py

```diff
@@ -13,12 +14,17 @@
         MANIFESTER_DIRECTORY = envar_location
 
 settings_path = MANIFESTER_DIRECTORY.joinpath("manifester_settings.yaml")
-
 validators = [
     Validator("offline_token", must_exist=True),
     Validator("simple_content_access", default="enabled"),
 ]
 settings = Dynaconf(
     settings_file=str(settings_path.absolute()),
     ENVVAR_PREFIX_FOR_DYNACONF="MANIFESTER",
+    validators=validators,
 )
+
+try:
+    settings.validators.validate()
+except:
+    print("i don't even know, dude")
```

Comment:
Are you expecting some validation to fail but the tool still work?

We diaper validation in robottelo because we haven't split the config for dynamic validation (yet) and its entirely possible that part of the validation will fail and be irrelevant to the test selection/execution.

Here you might want to just validate and let the exception fly, because the config is more limited and expected to be correct.